docs/en/sql-reference/functions/functions-index.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
Can we have this as index.md and the other file as overview.md  @Blargian for consistency?

Reply:
@gingerwizard There is some logic in naming them both as index.md. The docusaurus autogenerated sidebars pull all the markdown files in a given folder, and there is no easy way to tell it which files not to display. For trying to fix the breadcrumbs we don't want to display the overview files, but rather have them shown when the user clicks the top level dropdown menu. The reason for this is that in many places the overview file has the same name as the dropdown folder title, which seems unnecessary, like this one for instance:

![image](https://github.com/user-attachments/assets/95a96ee3-6065-4681-8348-dc23264856fe)

Here for instance the overview page should show when clicking 'Regular Functions' and the 'overview' doesn't need to be made a child:

![image](https://github.com/user-attachments/assets/b9310a7a-c2d8-44ec-afb0-d6a664e75b3b)
 
In order to fix that I have made this change in PR [2779](https://github.com/ClickHouse/clickhouse-docs/pull/2779) to customize the sidebar generator:

https://github.com/ClickHouse/clickhouse-docs/blob/04d6f55474b71ed0f2e3f27187654fdab2a14645/docusaurus.config.js#L60-L66

If it finds a file ending `index.md` then it will not display it as a link underneath the dropdown folder (but rather when clicking the folder itself).

Ideally the folder structure should match the web-page structure but for these functions we have a 'regular functions' grouping in the sidebar even though this grouping doesn't exist as a folder in the docs structure. I could place all the functions .md documents in a folder called 'regular functions' but then we need to change all the slugs which is a pain.